src/haddock/modules/analysis/rmsdmatrix/defaults.yaml

```diff
@@ -0,0 +1,10 @@
+max_models:
+  default: 10000
+  type: integer
+  min: 1
+  max: 20000
+  title: Maximum of models for rmsd matrix calculation
```

Comment:
suggestion: `Maximum number of models to calculate RMSD matrix`

Reply:
agreed!